test/jdk/javax/swing/JScrollPane/AcceleratedWheelScrolling/HorizScrollers.java

```diff
@@ -161,8 +161,9 @@ private void showFinalReminderIfNeeded(boolean isFailure) {
             if (scrollAmount != 3) {
                 JOptionPane.showMessageDialog(
                         ConfigPanel.this.getTopLevelAncestor(),
-                        ("Test %s. please make sure you have restored " +
-                                "the original speed value blah blah")
+                        ("Test %s. Please make sure you have set " +
+                                "the correct speed value and keep the " +
+                                "mouse stationary.")
```

Comment:
The updated instructions should be corrected as well.

It seems that this `JOptionPane` dialog is only displayed when one of the phases fails at non-default speed, and there is no further testing.
So the the `keep the mouse stationary` part is a little bit too late. 

This can be an issue though, but I think it is better to add a delay just before the test starts(`RTLScrollers.runTest`) to avoid catching mouse movements from a test used when he clicks the button.

Reply:
I see your point. Adjusted the dialog again to make sense for both pass/fail to restore the original speed.

Also added a `robot.delay` to the beginning of `runTest`.